index.bs

```diff
@@ -142,48 +142,52 @@ To <dfn>construct a <a>Geolocation Sensor</a> object</dfn> the user agent must i
 ### GeolocationSensor.latitude ### {#geolocationsensor-latitude}
 
 The {{GeolocationSensor/latitude!!attribute}} attribute of the {{GeolocationSensor}} interface
-represents the latitude coordinate of the [=geolocation=] in decimal degrees [[!WGS84]]. This
-attribute returns [=map/value=] for [=latest reading=]["latitude"] [=map/entry=]. Initially, its
-value must be null.
+returns the result of invoking [=get value from latest reading=] with `this` and "latitude" as
+arguments. Initially, its value must be null. It represents the latitude coordinate of the
+[=geolocation=] in decimal degrees [[!WGS84]].
 
 ### GeolocationSensor.longitude ### {#geolocationsensor-longitude}
 
 The {{GeolocationSensor/longitude!!attribute}} attribute of the {{GeolocationSensor}} interface
-represents the longitude coordinate of the [=geolocation=] in decimal degrees [[!WGS84]]. This 
-attribute returns [=map/value=] for [=latest reading=]["longitude"] [=map/entry=]. Initially, its
-value must be null.
+returns the result of invoking [=get value from latest reading=] with `this` and "longitude" as
+arguments. Initially, its value must be null. It represents the longitude coordinate of the
+[=geolocation=] in decimal degrees [[!WGS84]].
 
 ### GeolocationSensor.altitude ### {#geolocationsensor-altitude}
 
 The {{GeolocationSensor/altitude!!attribute}} attribute of the {{GeolocationSensor}} interface
-represents the altitude of the [=geolocation=] in meters above the WGS 84 ellipsoid [[!WGS84]].
-This attribute returns [=map/value=] for [=latest reading=]["altitude"] [=map/entry=]. Initially,
-its value must be null.
+returns the result of invoking [=get value from latest reading=] with `this` and "altitude" as
+arguments. Initially, its value must be null. It represents the [=geolocation=] in meters above
+the WGS 84 ellipsoid [[!WGS84]].
 
 ### GeolocationSensor.accuracy ### {#geolocationsensor-accuracy}
 
 The {{GeolocationSensor/accuracy!!attribute}} attribute of the {{GeolocationSensor}} interface
-represents the accuracy of the value of [=latest reading=]["longitude"] [=map/entry=] and
-value of [=latest reading=]["altitude"] [=map/entry=] in meters with a 95% confidence level.
-Initially, its value must be null.
+returns the result of invoking [=get value from latest reading=] with `this` and "accuracy" as
+arguments. Initially, its value must be null. It represents the accuracy of the
+[=latest reading=]["latitude"] value and [=latest reading=]["longitude"] value in meters with a
```

Comment:
That is interesting, what if `[=latest reading=]["latitude"]` is null? Same for `altitude`.

Reply:
The Geolocation API defines `latitude` and `longitude` to have non-null values, so dodges this issue.

For `altitudeAccuracy`, it states: "If the implementation cannot provide altitude information, the value of this attribute must be null." implying if `altitude` is null also `altitudeAccuracy` is null.

Would it be logical for both `accuracy` and `altitudeAccuracy` be null in such a case? I feel it'd be good for API ergonomics if latitude/longitude/altitude all would behave consistently in this regard.